msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-meeting-stage.md

```diff
@@ -0,0 +1,196 @@
+| **meetingChatTab** | A tab in the header of a group chat between a set of users for a scheduled meeting. You can specify either **meetingChatTab** or **meetingDetailsTab** to ensure the apps work in mobile. |
+| **meetingDetailsTab** | A tab in the header of the meeting details view of the calendar. You can specify either **meetingChatTab** or **meetingDetailsTab** to ensure the apps work in mobile. |
+| **meetingSidePanel** | An in-meeting panel opened through the unified bar (U-bar). |
+| **meetingStage** | An app from the `meetingSidePanel` can be shared to the meeting stage. You can't use this app either on mobile or Teams room clients. |
+
+#### Configure tab app for a meeting
+
+Apps in meetings can use the following contexts: `meetingChatTab`, `meetingDetailsTab`, `meetingSidePanel` and `meetingStage`. Once a meeting participant installs an app and configures the tab in meeting, all targeted other contexts of the app for the given meeting will start rendering the tab.
+
+The following code snippet is an example of a configurable tab used in an app for Teams meetings:
+
+```json
+
+"configurableTabs": [
+    {
+      "configurationUrl": "https://contoso.com/teamstab/configure",
+      "canUpdateConfiguration": true,
+      "scopes": [
+        "team",
+        "groupchat"
+      ],
+      "context":[
+        "channelTab",
+        "privateChatTab",
+        "meetingChatTab",
+        "meetingDetailsTab",
+        "meetingSidePanel",
+        "meetingStage"
+     ]
+    }
+  ]
+```
+
+---
+
+#### Other examples
+
+Default context for tabs (if not specified) is:  
+
+```json
+"context":[ 
+  "channelTab", 
+  "privateChatTab", 
+  "meetingChatTab", 
+  "meetingDetailsTab" 
+     ] 
+```
+
+To prevent an app from showing in non-meeting group chats, you would set the following context on their tab:
+
+```json
+"context":[ 
+  "meetingSidePanel", 
+  "meetingChatTab", 
+  "meetingDetailsTab" 
+     ] 
+```
+
+To target just the in-meeting side panel experience:  
+
+```json
+"context":[ 
+  "meetingSidePanel” 
+     ] 
+```
+
+### Advance Tab SDK APIs
+
+The Microsoft Teams JavaScript client SDK is a rich SDK used to create Tabs using Javascript. Use latest TeamsJS (V.2.0 or later) to work in Teams, Office, and Outlook. For more information, see [Teams JavaScript client SDK](/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?branch=pr-en-us-6722&tabs=javascript%2Cmanifest-teams-toolkit).
+
+### Frame context
+
+Microsoft Teams JavaScript library exposes the frameContext in which your meeting tab URL is loaded in the getContext API. The possible values of frameContext are content, task, setting, remove, sidePanel. This allows you to build customized experiences according to the size of the frame in which your app is loaded. For more information, see [getContext API](/microsoftteams/platform/tabs/how-to/access-teams-context?tabs=teamsjs-v2).
+
```

Comment:
A user reading this might have no understanding of what a frameContext is. We say we expose a frameContext but that could be anything.

Reply:
> The possible values of frameContext are content, task, setting, remove, sidePanel.

This is not a complete list, from the SDK these are possible: settings, content, authentication, remove, task, sidePanel, stage, meetingStage.

Are these the ones that are listed only possible in meeting apps?

Reply:
> This allows you to build customized experiences according to the size of the frame in which your app is loaded. 

Size is not how I would explain this to user. You don'e need frameContext to change the experience based on size (you can just use media quries for that). 

Instead I would say "This allow you to build customised experienecs based on where the app is rendering. e.g. Showing a specific collaboration focused UI when in the `meetingStage` and a different meeting prepartion UI in the chat tab (`content`)

Reply:
Lastly,

> For more information, see [getContext API](/microsoftteams/platform/tabs/how-to/access-teams-context?tabs=teamsjs-v2).

There is not much "more" information on frameContext at that URL, there is more information on getContext though, would it be better to call that out instead?